Competition2017/src/competition/subsystems/autonomous/DriveToHopperThenBoilerCommandGroup.java

```diff
@@ -18,17 +19,20 @@
     public DriveToHopperThenBoilerCommandGroup(XPropertyManager propManager,
             Provider<DriveForDistanceCommand> driveForDistanceProvider,
             Provider<RotateToHeadingCommand> rotateToHeadingProvider,
-            Provider<DriveToHopperCommandGroup> driveToHopperCommandGroupProvider) {
+            Provider<DriveToHopperCommandGroup> driveToHopperCommandGroupProvider,
+            ShiftGearCommand shiftCommand) {
         headingToFaceBoiler = propManager.createPersistentProperty("Heading To Face Boiler In Degees", 70.2);
         distanceFromTurningPointToBoiler = propManager
                 .createPersistentProperty("Distance From Turning Point To Boiler In Inches", 72);
         distanceBackFromHopperToTurningPoint = propManager
                 .createPersistentProperty("Distance Back From Hopper To Turning Point", -43.25);
 
+        shiftCommand.setGear(Gear.LOW_GEAR);
+        this.addSequential(shiftCommand);
         
         // drive to hopper from wall
         DriveToHopperCommandGroup driveToHopper = driveToHopperCommandGroupProvider.get();
-        this.addSequential(driveToHopper);
+        this.addSequential(driveToHopper, 0.1);
```

Comment:
Fixed by https://github.com/Team488/TeamXbot2017/pull/128; we really need to be more careful in reviews!